pkg/discovery/dtofactory/node_roles_group_dto_builder.go

```diff
@@ -0,0 +1,92 @@
+package dtofactory
+
+import (
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/turbonomic/kubeturbo/pkg/discovery/detectors"
+	"github.com/turbonomic/kubeturbo/pkg/discovery/repository"
+	"github.com/turbonomic/kubeturbo/pkg/discovery/util"
+	"github.com/turbonomic/turbo-go-sdk/pkg/builder/group"
+	"github.com/turbonomic/turbo-go-sdk/pkg/proto"
+)
+
+type NodeRolesGroupDTOBuilder struct {
+	cluster  *repository.ClusterSummary
+	targetId string
+}
+
+func NewNodeRolesGroupDTOBuilder(cluster *repository.ClusterSummary,
+	targetId string) *NodeRolesGroupDTOBuilder {
+	return &NodeRolesGroupDTOBuilder{
+		cluster:  cluster,
+		targetId: targetId,
+	}
+}
+
```

Comment:
Add comment to the method regarding when the group or a policy is created for different nodes.